src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,18 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e:
+                    msg = ''
+                    for authzr in e.failed_authzrs:
+                       msg += f'\nAuthorization for identifier {authzr.body.identifier} failed.'
+                       msg += '\nHere are the challenges that were not fulfilled:'
+                       for challenge in authzr.body.challenges:
+                         msg += \
+                               f'\nChallenge Type: {challenge.chall.typ}' \
+                               f'\nError information:' \
+                               f'\nType: {challenge.error.typ if challenge.error else ""}' \
+                               f'\nDetails: {challenge.error.detail if challenge.error else ""}\n\n'
```

Comment:
Reading this again, this will do not look OK:

```
Challenge Type: foo
Error Information:
Type: errortype
Details: errordetail
```

It will look like `Error Information` is a field with a missing/null/empty value.

Reply:
Ahh shoot, because it's an fstring?

Reply:
No, not about f-string but actual text visualization.
Need any kind of representation so its clear type and details are subitems of `Error information`.

Perhaps `-`.

```
Error Information:
 - Type: ...
 - Details: ...
```